openrr-gui/src/velocity_sender.rs

```diff
@@ -422,6 +428,11 @@ where
         Command::none()
     }
 
+    fn subscription(&self) -> iced::Subscription<Self::Message> {
+        iced_futures::backend::native::tokio::time::every(std::time::Duration::from_millis(100))
+            .map(messenger)
```

Comment:
I don't know the details, but it seems lambda works fine, right?

```suggestion
            .map(|| Message::Retained)
```

Reply:
Yes.  I updated it.